### PR TITLE
[v5.1] update k8s backup page

### PR DESCRIPTION
**Pull request by gggina**

### CODE REVIEW

File: setup/data-management/backup_restore/kubernetes_backup.md [hunk 1]
```diff
@@ -322,18 +329,59 @@ job.batch "elasticsearch-list-snapshots-20210224t133115" deleted
...
+   kubectl scale --replicas=0 deployment/mm2es
+   kubectl scale --replicas=0 deployment/trace2es
+   ```
+
+2. Open a port-forward to the Elasticsearch master:
+
+   ```bash
+   kubectl port-forward service/stackstate-elasticsearch-master 9200:9200
+   ```
+
```
Comment (viliakov) [inline setup/data-management/backup_restore/kubernetes_backup.md:350]: How about adding a command to get the list of the indices?
```
curl "http://localhost:9200/_cat/indices?v=true"
```

the output

```
green open sts_internal_events-2022.09.20 fTk7iEYtQI6ruVFuwNnbPw 1 0  125 0  71.7kb  71.7kb
health status index                          uuid                   pri rep docs.count docs.deleted store.size pri.store.size
green  open   sts_internal_events-2022.09.20 fTk7iEYtQI6ruVFuwNnbPw   1   0        125            0     71.7kb         71.7kb
green  open   .geoip_databases               GYA_c3i6QKenfFehnwBcAA   1   0         41            0     38.8mb         38.8mb
green  open   sts_multi_metrics-2022.09.20   MT1DceQPTDWVIfBJdl7qUg   3   0       1252            0    550.1kb        550.1kb
```

File: setup/data-management/backup_restore/kubernetes_backup.md [hunk 2]
```diff
@@ -322,18 +329,59 @@ job.batch "elasticsearch-list-snapshots-20210224t133115" deleted
...
 
 {% hint style="danger" %}
-When a snapshot is restored, existing indices will NOT be overwritten. Use the Elasticsearch [delete index API \(elastic.co\)](https://www.elastic.co/guide/en/elasticsearch/reference/7.6/indices-delete-index.html) to remove them first. 
+**When a snapshot is restored, existing indices will NOT be overwritten.** Use the Elasticsearch [delete index API \(elastic.co\)](https://www.elastic.co/guide/en/elasticsearch/reference/7.6/indices-delete-index.html) to remove them first. 
 
 See [delete Elasticsearch indices](kubernetes_backup.md#delete-elasticsearch-indices).
 {% endhint %}
 
-To restore an Elasticsearch snapshot, select a snapshot name and pass it as the first parameter in the following command line:
+To restore an Elasticsearch snapshot, select a snapshot name and pass it as the first parameter in the following command line. You can optionally specify a second parameter with the indices that should be restored. If not specified, all indicies that match the Helm value `backup.elasticsearch.scheduled.indices` will be restored (default `"sts*"`):
```
Comment (viliakov) [inline setup/data-management/backup_restore/kubernetes_backup.md:379]: We may clarify that the second parameter might accept comma-separeted list of the indices, for example, "index1,index2"

**Review by viliakov — APPROVED** [review]
LGTM

File: setup/data-management/backup_restore/kubernetes_backup.md (hunk 1 above)
Comment (gggina) [reply setup/data-management/backup_restore/kubernetes_backup.md:350]: done

**Pull request merged**